ci/test_wheel_dask_cudf.sh

```diff
@@ -16,4 +16,6 @@ python -m pip install git+https://github.com/dask/dask.git@2023.9.2 git+https://
 # echo to expand wildcard before adding `[extra]` requires for pip
 python -m pip install $(echo ./dist/dask_cudf*.whl)[test]
 
+# Run tests in dask_cudf/tests and dask_cudf/io/tests
 python -m pytest -n 8 ./python/dask_cudf/dask_cudf/tests
+python -m pytest -n 8 ./python/dask_cudf/dask_cudf/io/tests
```

Comment:
```suggestion
python -m pytest -n 8 ./python/dask_cudf/dask_cudf/
```

Should we rather do this?

cc: @vyasr if you have any thoughts on this

Reply:
Right - Was about to comment on this. Seems fine to me either way, so I'll go with whatever others like.

Reply:
Let's adopt Prem's suggestion and rely on pytest discovery.